build_region.R

```diff
@@ -99,14 +106,8 @@ l$time_fast <- rf$time
 l$time_quiet <- rq$time
 l$cirquity <- rf$length / l$dist
 l$distq_f <- rq$length / rf$length
-l$avslope <- rf$av_incline
-l$co2_saving <- rf$co2_saving
-l$calories <- rf$calories
-l$busyness <- rf$busyness
-l$avslope_q <- rq$av_incline
-l$co2_saving_q <- rq$co2_saving
-l$calories_q <- rq$calories
-l$busyness_q <- rq$busyness
+l$avslope <- rf$av_incline * 100
+l$avslope_q <- rq$av_incline * 100
```

Comment:
Anna asked for this specifically.  I'm slightly worried but after a search I can't see it effecting anything.